### PR TITLE
Edits to pair dipole doc page

### PR DESCRIPTION
**Pull request by sjplimp**
**Summary**

Update the pair dipole doc page for potentials which include dipolar interactions.  Some parts were out-of-date.  Some parts were unclear.

**Related Issue(s)**

N/A

**Author(s)**

Steve

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

**Implementation Notes**

**Post Submission Checklist**

- [ ] The feature or features in this pull request is complete
- [ ] Licensing information is complete
- [ ] Corresponding author information is complete
- [ ] The source code follows the LAMMPS formatting guidelines
- [ ] Suitable new documentation files and/or updates to the existing docs are included
- [ ] The added/updated documentation is integrated and tested with the documentation build system
- [ ] The feature has been verified to work with the conventional build system
- [ ] The feature has been verified to work with the CMake based build system
- [ ] Suitable tests have been added to the unittest tree.
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

**Further Information, Files, and Links**

### CODE REVIEW

**Comment by sjplimp** [comment]
@akohlmey I'm having a collaborator read this to make sure it is now more clear - so hold off on merging.

**Comment by akohlmey** [comment]
> @akohlmey I'm having a collaborator read this to make sure it is now more clear - so hold off on merging.

@sjplimp the easiest way to make it impossible to merge an incomplete pull request is to submit it as a "Draft pull request" (the "submit pull request" button has a little triangle where it turns into a drop down list and you can choose.
Also you can do this after the fact, by clicking on the "convert to draft" link right underneath the list of reviewers.

Such a draft pull request should not be reviewed until you have clicked on the "Ready for Review" button and merging is prohibited until then.

**Comment by sjplimp** [comment]
done - I though labeling it work-in-progress also prevented merging

**Comment by akohlmey** [comment]
> done - I though labeling it work-in-progress also prevented merging

Only by convention. We used the label before GitHub implemented "Draft Pull requests".
Besides blocking a merge, submitting the pull request as draft also has the advantage that people listed as "Code owners" will not be notified until you declare the PR ready for review. Of course this does not apply when you convert back to draft later...

**Comment by sjplimp** [comment]
@akohlmey this is ready to merge after Stan reviews it

**Review by stanmoore1 — APPROVED** [review]
I approve.

**Pull request merged**
